packages/common/container-definitions/src/test/types/validateContainerDefinitionsPrevious.ts

```diff
@@ -325,30 +325,6 @@ declare function use_old_InterfaceDeclaration_ICodeDetailsLoader(
 use_old_InterfaceDeclaration_ICodeDetailsLoader(
     get_current_InterfaceDeclaration_ICodeDetailsLoader());
 
-/*
-* Validate forward compat by using old type in place of current type
-* If breaking change required, add in package.json under typeValidation.broken:
-* "InterfaceDeclaration_ICodeLoader": {"forwardCompat": false}
-*/
-declare function get_old_InterfaceDeclaration_ICodeLoader():
```

Comment:
was this file change manually? can you try reverting this file changes and do a local build again?

test/types/** files are auto generated on build